ni/nimi-python
# Update metadata to include all information, no longer require addon

- [X] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).
- [X] I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable.
- [X] I've added and updated tests applicable for this pull request
### What does this Pull Request accomplish?
* Update metadata so that main file has all information required
    * Exceptions
        * Package version - specific to GitHub and not based on underlying driver version
        * Latest driver version tested
* Use actual booleans instead of strings that look like booleans: `False` vs `'False'`
* Add support for array behavior 'ivi-dance-with-a-twist'
    * Requires two calls to function similar to 'ivi-dance'
    * Instead of the size being returned in the function return, it is returned in a separate output parameter
    * nidigital will use it
* Add nidigital metadata - but do not build it yet - See #991

### ~~List issues fixed by this Pull Request below, if any.~~

### What testing has been done?
* Visual inspection
* Unit tests
